docs/design/2018-10-08-online-DDL.md

```diff
@@ -0,0 +1,75 @@
+# TiDB DDL architecture
+- Author(s):     [zimuxia](https://github.com/zimulala) (Xia Li)
+- Last updated:  2018-10-08
+## Overview 
+This article will describe the architecture of the Online DDL implemented in TiDB.
+## Background
+Many databases lock tables when performing DDL operations. During this time, many transactions involved in this table are blocked (some databases support reading operations, but at the expense of consuming a large amount of memory). The larger the table is, the longer it affects the transactions.
+To simplify the design, the entire system allows only one node to make schema changes at the same time.
```

Comment:
Add a blank line before this sentence. 

`To simplify the design, in TiDB, the entire system allows only one node to make schema changes at the same time.`

Reply:
Done.